### ISSUE TITLE
Now no error, but not downloading

### ISSUE THREAD

**Issue by horizon-io** [codecoding/PluralSightDownloader#39]
Hello,
I tried the last version of the program (yesterday's) it shows no error, but it doesn't download anything,
in status bar it shows the message "Download completed" and the **progress bar** is moving and there's disk and network activity (seen in Task Manager) but there's no file saved on the download path.

Should i provide more info?

**Comment by robertohuertasm** [comment]
HI @horizon-io,

are you experiencing this when downloading a specific course? Maybe just a module? I've tried to reproduce your use case but no luck. It's always working. Obviously, there's something that I'm missing. 

Do you check Organize in Folders when you start the download?

**Comment by robertohuertasm** [comment]
Just to get sure you're using the latest version. Do you see 30000 or 10000 in the delay textbox?

**Comment by horizon-io** [comment]
thank you @robertohuertasm 
I use the last version, I see 30000 in delay.
All courses like this, all modules, even clips, Organize in folders checked or not the same result.

**Comment by robertohuertasm** [comment]
Have you tried reinstalling from the scratch? Which OS version are you using? Have you tried changing the path?

**Comment by horizon-io** [comment]
Dear @robertohuertasm 
Yesterday I reinstalled but the same problem,
I always change the path of the download, but nothing.
I use Windows 10.

**Comment by robertohuertasm** [comment]
Do you have the chance of installing it into another computer?

**Comment by horizon-io** [comment]
Hi Mr. @robertohuertasm, I am trying to find another PC and another ISP to test it.

**Comment by robertohuertasm** [comment]
I think I've managed to reproduce it... It somehow depends on the type of account but I'm not really sure why it's happening.

It says downloading but the progress bar remains blank, doesn't it?

**Comment by robertohuertasm** [comment]
Well, it's going to sound a little dumb but have you checked that your subscription is ok? I mean, you may log into the site but you're not paying your subscription so you won't be able to download anything. The case that I was describing before was precisely this case.

Please let me know!

**Comment by horizon-io** [comment]
@robertohuertasm 
It says download completed, but progress bar still in progress.
--------
My Subscription will end late this month.
btw, the subscriptions is offered by Microsoft Dev Essentials (not trial), I think it's the Plus one, I can download course to mobile, with no problem (but I am not very comfortable with this).
I used to download using your app before, I've downloaded 2 courses, and some partially download, and after an update, I could not download anything.

**Comment by horizon-io** [comment]
@robertohuertasm 
UPDATE: Well, in the old version it was showing that my country is blocked by the US laws...etc,
I used vpn, but it was limited and bad, most of time causes to crash the app (because disconnects),
In new version it doesn't show that I am blocked, but just not downloading.
My new ISP didn't let me to use the VPN I used before, so I've just find another good vpn that can work with my ISP, and we are ok now.
I am downloading with VPN.

Thanks a lot Mr. Roberto, your app helping me a lot.
